content/docs/Konveyor/Kai/IDESettings/proc_configuring-solution-server-settings-file.md

```
@@ -0,0 +1,46 @@
+---
+title: "Configuring the Solution Server settings"
+date: 2025-12-03T14:58:52-06:00
+draft: false
+---
+You need a Keycloak realm and the Solution Server URL to connect
+Konveyor extension with the Solution Server.
+
+**Prerequisites**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)